tests/integration/test_spm_parameterisations.py

```diff
@@ -241,16 +240,17 @@ def test_multiple_signals(self, multi_optimiser, two_signal_cost):
         }
 
         if multi_optimiser is pybop.SciPyDifferentialEvolution:
-            common_args["bounds"] = [[0.375, 0.775], [0.375, 0.775]]
+            common_args["bounds"] = {"lower": [0.375, 0.375], "upper": [0.775, 0.775]}
             if isinstance(two_signal_cost, pybop.GaussianLogLikelihood):
-                common_args["bounds"].extend([[0.0, 0.05], [0.0, 0.05]])
+                common_args["bounds"]["lower"].extend([0.0, 0.0])
+                common_args["bounds"]["upper"].extend([0.05, 0.05])
 
         # Test each optimiser
         optim = multi_optimiser(**common_args)
 
         # Add sigma0 to ground truth for GaussianLogLikelihood
         if isinstance(two_signal_cost, pybop.GaussianLogLikelihood):
-            self.ground_truth = np.concatenate((self.ground_truth, combined_sigma0))
+            self.ground_truth = np.concatenate((self.ground_truth[:2], combined_sigma0))
```

Comment:
Likewise to the above:
```suggestion
            self.ground_truth = np.concatenate((self.ground_truth, combined_sigma0))
```